packages/web/components/twitter-section/twitter-section.tsx

```diff
@@ -31,7 +31,7 @@ function Tweet({ tweetContent }: TweetProps) {
   return (
     <div className="flex flex-col items-start gap-4 self-stretch py-3">
       <div className="flex-start flex gap-4 self-stretch 1.5xs:flex-col">
-        <div className="flex h-[52px] w-[52px] items-center gap-3 rounded-full bg-white-high 1.5xs:self-center" />
+        <div className="flex h-tweetavatar w-tweetavatar items-center gap-3 rounded-full bg-white-high 1.5xs:self-center" />
```

Comment:
I don't like that name, just add it inside "spacing" config, and you can call it "13", like tailwind do. Spacing values are also used for width and height.

Reply:
gotcha